src/export_json.lean

```diff
@@ -340,20 +340,52 @@ do docs ← olean_doc_strings,
      (filename, json.array $ l.map write_module_doc_pair) :: rest
    end) [])
 
-meta def get_instances : tactic (rb_lmap string string) :=
+/-- Extract `[foo, bar]` from `has_pow foo bar`.
```

Comment:
This function will also extract the implicit arguments from the typeclass, which results in all kinds of spurious instances on real.

Reply:
How about this?
```lean
meta def get_explicit_args (e : expr) : tactic (list expr) :=
fold_explicit_args e [] (λ xs x, pure $ xs ++ [x])
```

Reply:
Excellent, I didn't know about `fold_explicit_args`